ruby/ql/lib/codeql/ruby/frameworks/Files.qll

```diff
@@ -189,12 +189,6 @@ module IO {
     }
   }
 
-  // "Direct" `IO` instances, i.e. cases where there is no more specific
-  // subtype such as `File`
-  private class IOInstanceStrict extends IOInstance {
```

Comment:
@alexrford you added this class originally. Is its deadness unintentional?

Reply:
Yeah, I should have removed this as part of the refactoring in https://github.com/github/codeql/pull/8138